content/tutorials/quickstart/create-accounts-send-xrp.md

```diff
@@ -28,7 +28,7 @@ To get started, create a new folder on your local disk and install the JavaScrip
     npm install xrpl
 ```
 
-Download and expand the [Quickstart Samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/js/quickstart.zip){.github-code-download} archive.
+Download and expand the [Quickstart Samples]([https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/js/quickstart.zip](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/js)){.github-code-download} archive.
```

Comment:
It looks like some kind of copy-paste glitch happened here? (Perhaps you selected text and pasted a URL, expecting it to be replaced, but the GitHub editor interpreted that as trying to make the selected text a link?)

Guessing it should look like this instead:

```suggestion
Download and expand the [Quickstart Samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/js/){.github-code-download} archive.
```